File: services/org-svc/src/main/java/com/supportme/pay/orgsvc/careers/JobPositionRepository.java
package com.supportme.pay.orgsvc.careers;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface JobPositionRepository extends JpaRepository<JobPosition, Long> {
    List<JobPosition> findByOrganizationIdOrderBySortAscIdAsc(Long organizationId);

    List<JobPosition> findByOrganizationIdAndActiveTrueOrderBySortAscIdAsc(Long organizationId);
}

File: pay-backend/storefront-domain/src/main/kotlin/com/supportme/pay/storefront/domain/repository/JobPositionRepository.kt
package com.supportme.pay.storefront.domain.repository

import com.supportme.pay.storefront.domain.entity.JobPosition
import org.springframework.data.jpa.repository.JpaRepository

interface JobPositionRepository : JpaRepository<JobPosition, Long> {
    fun findAllByActiveTrueOrderBySortAscIdAsc(): List<JobPosition>
}
